test/unit/utils.ts

```diff
@@ -84,3 +84,7 @@ export function responseFrom(data: any, status: number = 200, headers: any = {})
     text: JSON.stringify(data),
   };
 }
+
+export function errorFrom(data: any, status: number = 500): HttpError {
```

Comment:
If I recall, the default error http status for majority of errors above is 400. It is 500 here.

Reply:
I don't think it matters that much. The old test cases did not use any status codes, which implies that the auth module is agnostic to the actual HTTP status codes.

With this change we now have test cases that return both 500 and 400 errors. WDYT?

Reply:
In the past there was no indication of http status code during test simulation. With this change, it is implied all simulated status codes are 500s. It is true Auth doesn't care in this case, but other services may. Note this could also change in the future. 500 could be handled differently.
Anyway, this is probably fine for now.